lib/src/Uncertainty/Algorithm/OrthogonalBasis/OrthogonalProductPolynomialFactory.cxx

```diff
@@ -154,6 +154,70 @@ String OrthogonalProductPolynomialFactory::__repr__() const
          << " measure=" << measure_;
 }
 
+String OrthogonalProductPolynomialFactory::__str__(const String & /*offset*/) const
+{
+  return __repr_markdown__();
+}
+
+/* String formatter with blank space right padding */
+String OrthogonalProductPolynomialFactory::padString(const String & string, const UnsignedInteger columnWidth) const
```

Comment:
Is this padding function really different from the previous one?

Reply:
Not. But where can I write it then?

Reply:
In String.hxx as a static method

Reply:
In OSS.hxx actually. Fixed at fc0b6fdd8ded654790aaebae44f23c4ad4cc1d2f